gcloud-java-examples/README.md

```diff
@@ -33,7 +33,7 @@ To run examples from your command line:
 
 1. Login using gcloud SDK (`gcloud auth login` in command line)
 
-2. Set your current project using `gcloud config set project PROJECT_ID`
+2. If you are running an example other than `ResourceManagerExample`, set your current project using `gcloud config set project PROJECT_ID`. If you are running `ResourceManagerExample`, unset your project by using `gcloud config unset project`.
```

Comment:
is it a problem if you project id is set also for resource manager example? If not I would just say something like "(this step is not necessary for ResourceManagerExample)".